examples/ReorderExample.js

```diff
@@ -66,7 +66,7 @@ class ReorderExample extends React.Component {
     } else {
       if (fixedColumns.indexOf(event.reorderColumn) !== -1) {
         columnOrder.splice(fixedColumns.length - 1, 0, event.reorderColumn)
-      } else {
```

Comment:
why was this change necessary?

Reply:
My bad, this is leftover from some troubleshooting. Reverting.

Reply:
Don't forget this one

Reply:
Yup, forgot to push.